### DISCUSSION TITLE
Table files documented as optional/automatic generation needed to be created

### DISCUSSION THREAD

**Discussion by ericwenger-pm** [nih-cfde/published-documentation#207] [Q&A] [answered]
A finding we came across during c2m2 submit tool testing was that the table tsv files noted as optional at https://github.com/nih-cfde/published-documentation/wiki/C2M2-Table-Summary ("Automatically created by submission software") actually needed to be created and submitted in order to submit the manifest successfully.

Here are the files we found we needed to create:
anatomy.tsv
assay_type.tsv
data_type.tsv
file_format.tsv
ncbi_taxonomy.tsv

Just calling this out either that the documentation be updated or that functionality be fixed.

**Comment by dankolbman** [comment]
This is possibly because we tried to submit without these tables in the directory but still being listed in our datapackage.json.
It would be nice to understand how we should generate/obtain the datapackage.json as it seems like we could completely make it up thus allowing successful validation of anything.

**Comment by ACharbonneau** [comment, marked as answer]
It sounds like you are skipping the helper script. I think this page should help: https://github.com/nih-cfde/published-documentation/wiki/Building-a-datapackage
